src/virtualenv/util/path/_sync.py

```diff
@@ -56,7 +56,7 @@ def copy(src, dest):
 def copytree(src, dest):
     for root, _, files in os.walk(src):
         dest_dir = os.path.join(dest, os.path.relpath(root, src))
-        if not os.path.exists(dest_dir):
+        if not os.path.isdir(dest_dir):
```

Comment:
should only check that it's a directory, not in general

there's also a TOCTOU bug here but it's unlikely to be hit